website/content/rustdoc.md

```diff
@@ -0,0 +1,5 @@
++++
+title = "Rustdoc"
+template = "rustdoc.html"
+weight = 1
```

Comment:
What do these weight values do?

Reply:
They define the order of the nav items.
Lowest is farthest left.